server/chat-plugins/helptickets.js

```diff
@@ -82,6 +97,19 @@ class HelpTicket extends Rooms.RoomGame {
 		this.ticket = ticket;
 		/** @type {string[]} */
 		this.claimQueue = [];
+		/* Stats */
+		/** @type {string[]} */
+		this.involvedStaff = [];
```

Comment:
This can now be `ID[]` because `User#userid` is `ID`!

Reply:
Also, why not just use a set? It would remove the logic needed to check if a staff member is already there.